how-to-docs/chisel.md

```diff
@@ -0,0 +1,82 @@
+#
+## Providing Access to Third Party Services (WIP)
+#
+
+> *The Problem*: we want to provides services running in ACP access to the third party services as well as the ability to have user-based access controls. At present network access in ACP is provided via Calico, but this becomes redundent when the traffic egresses the cluster. So simply peering networks together either through VPC peering or VPN connections doesn't provide the controls we want. We could rely on user-authentication on third-party service but not all services are authenticated (take POISE) and beyond that peering networks provide's no means of auditing traffic that is traversing the bridged networks.
```

Comment:
we want to `provide` services...  
tyops: `redundent` -> `redundant`, `provide's` -> `provides`